utils/auth/common/authLevel.go

```diff
@@ -10,6 +14,34 @@ const (
 	Attendee
 	// Volunteer is the auth level that represents a user who has access to volunteer features
 	Volunteer
-	// Organizer is the auth level that represents a user who has access to all features
-	Organizer
+	// Organiser is the auth level that represents a user who has access to all features
+	Organiser
```

Comment:
Since this has been renamed to `organiser`, don't we need to update all the references to this auth level.
For example in `routers/frontend/router.go` line 79
<img width="630" alt="Screenshot 2020-02-26 at 01 52 30" src="https://user-images.githubusercontent.com/12220760/75304233-ba969680-583a-11ea-9435-f06f040431ef.png">

Reply:
The line you're referring to was added in the branch that I just merged into master, the auth level was still called `Organizer` on that branch.

I've rebased and fixed this line. We shouldn't really have any hidden problems with this as the auth level `Organizer` no longer exists and if it's still being used anywhere in the code, the app will simply not compile and the tests will fail.